test/integration/targets/docker_network/tasks/tests/basic.yml

```diff
@@ -0,0 +1,117 @@
+---
+- name: Registering container name
+  set_fact:
+    cname_1: "{{ name_prefix ~ '-container-1' }}"
+    cname_2: "{{ name_prefix ~ '-container-2' }}"
+    cname_3: "{{ name_prefix ~ '-container-3' }}"
+    nname_1: "{{ name_prefix ~ '-network-1' }}"
+    nname_2: "{{ name_prefix ~ '-network-2' }}"
+- name: Registering container name
+  set_fact:
+    cnames: "{{ cnames }} + [cname_1, cname_2, cname_3]"
```

Comment:
Isn't `cnames: "[cname_1, cname_2, cname_3]"` sufficient ? Then `set_fact: cnames` at beginning of `main.yml` could be removed.

Reply:
Now there's only one sub-test (basic.yml), so yes, that would work, but as soon as one more is added, this is necessary (so that all names are collected, and the objects will be destroyed even in case of failing tests and other errors).

Reply:
Not sure what this future test will look like :)